semver.md

```diff
@@ -319,6 +319,24 @@ Abbreviating "version" as "v" is often seen with version control. Example:
 `git tag v1.2.3 -m "Release version 1.2.3"`, in which case "v1.2.3" is a tag
 name and the semantic version is "1.2.3".
 
+### What does SemVer consider a package?
+
+With "package" SemVer describes deployment units, independent of the actual
+technology used. Packages could be built from source artifacts or assemble
+binary artifacts.
+
+An actual eco-system might have a different terminology. For example In *Java*
```

Comment:
[Ecosystem](http://www.dictionary.com/browse/ecosystem?s=t) is one word, no need for the hyphen.

Reply:
Thanks @jwdonahue  - pushed a new version.

Reply:
Every morning I seem to see something new. Let's accumulate some comments for a few days and see where it lands.  I think we're almost there.  The second paragraph looks fine to me, but I nearly dislocated a cog on the last sentence in the first paragraph. Either the paragraph needs another sentence or ... I don't know yet.  I need to let it ferment.